examples/run_smc_on_freesolv.py

```diff
@@ -32,32 +55,90 @@ def parse_options():
     return cmd_args
 
 
-def save_smc_result(uid, smc_result, save_full_trajectories=False):
+def get_ff(ff_path=DEFAULT_FF) -> Forcefield:
+    """
+    Load the forcefield given the path to the ff py file.
+    """
+    return Forcefield.load_from_file(ff_path)
+
+
+def get_result_path(path: Path, mol_id: str) -> Path:
+    """
+    Return the path to the smc results for a particular molecule.
+    """
+    return path / f"summary_smc_result_{mol_id}.pkl"
+
+
+def get_full_traj_path(path: Path, mol_id: str) -> Path:
+    return path / f"full_smc_traj_{mol_id}.pkl"
+
+
+def save_smc_result(path: Path, mol, smc_result: Dict, cmd_args: argparse.Namespace, save_full_trajectories=False):
+    """
+    Save the smc results as a pkl'd dictionary.
+
+    Parameters
+    ----------
+    path:
+        Path to the smc results pkl cache
+
+    mol: ROMol
+        Molecule sampled using smc
+
+    smc_result:
+        'initial_samples_refined, 'final_samples_refined' correspond to
+        the refined samples from the end states.
+        'initial_log_weights', 'final_log_weights' correspond to the
+        log weights from both states.
+        'traj' is a list if trajectories, one for each lambda.
```

Comment:
nit: if -> of typo, I think the description is transposed? (there's a list of samples for each* lambda, but each of those lists is not time ordered. (*all but one lambda))